### PR TITLE
Improve stellar flux extraction and add new Correction using external data

### PR DESCRIPTION
**Pull request by PabloCorcho**
This PR includes refactoring of the methods used for extracting the flux of standard stars, a few small bugfixes on some corrections modules (e.g. astrometry) and the implementation of a new correction module that uses external imaging data to perform astrometry corrections (potentially also absolute flux calibrations).

### CODE REVIEW

**Comment by PabloCorcho** [comment]
@paranoya please have a look at these changes.

File: src/pykoala/corrections/flux_calibration.py [hunk 1]
```diff
@@ -239,105 +236,152 @@ def extract_stellar_flux(self, data_container,
...
             # Computing the curve of growth
             ###################################################################
-            x0, y0 = centre_of_mass(slice_data, x, y)
-            y_min, y_max = y.min(), y.max()
-            x_min, x_max = x.min(), x.max()
+            x0, y0 = centre_of_mass(slice_data * mask, x, y)
             # Make the growth curve
-            r2, growth_c = growth_curve_1d(slice_data, x - x0, y - y0)
+            r2 = ((x - x0)**2 + (y - y0)**2) * 3600**2  # expressed in arcsec^2
+            r2_dummy = np.arange(0, 10**2, 0.5**2)
```
Comment (paranoya) [inline src/pykoala/corrections/flux_calibration.py:298]: I guess it's OK, but why 10 arcsec?
It also feels a bit hard-coded...

**Review by paranoya — APPROVED** [review]
As always, my review was quick, but everything looks OK

File: src/pykoala/corrections/flux_calibration.py (hunk 1 above)
Comment (PabloCorcho) [reply src/pykoala/corrections/flux_calibration.py]: I completely missed that. I has been removed and added as an argument in the method with the same default values.

**Pull request merged**